starwars/src/components/cardComponent.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const DIV = styled.div`
```

Comment:
Great use of styled components! To make it even better you can nest you `H1` styles in the `DIV` styles. Just like how you would have done it in `LESS`